tests/redirection_tests.c

```diff
@@ -5,101 +5,143 @@
 #include <parser/parser.h>
 #include <parser/parse_redirection.h>
 #include "ctest.h"
+#include <fcntl.h>
 
-t_files files;
+#define BUFFERSIZE 1024
 
-CTEST(redirection_tests, basic_infile)
+CTEST(create_files, create_test_files)
 {
-	const char	*str = "   < infile cat -e";
-	char		buffer[4096];
-
-	init_files(&files);
-	files = get_redirection(&str);
-	ft_strlcpy(buffer, files.in, get_cmd_len(files.in) + 1);
-	ASSERT_STR("infile", buffer);
-	ASSERT_STR(" cat -e", str);
+    char no_implicit_quotes[] = "      test_file     ";
+    int test_fd = open(&no_implicit_quotes[0], O_RDWR | O_CREAT | O_TRUNC, 0664);
+    close(test_fd);
+    test_fd = open("test_file",  O_RDWR | O_CREAT | O_TRUNC, 0664);
 }
 
-CTEST(redirection_tests, basic_outfile)
+CTEST(spaces_test, empty_str)
 {
-	const char	*str = " > outfile | next cmd";
-	char		buffer[4096];
-	
-	init_files(&files);
-	files = get_redirection(&str);
-	ft_strlcpy(buffer, files.out, get_cmd_len(files.out) + 1);
-	ASSERT_STR("outfile", buffer);
-	ASSERT_STR(" | next cmd", str);
+    ASSERT_EQUAL(0, count_consecutive_spaces(""));
 }
 
-CTEST_DATA(command_table_redirection_tests)
+CTEST(spaces_test, spaced_str)
 {
-    t_command command_table[100];
-};
+    ASSERT_EQUAL(4, count_consecutive_spaces("    "));
+    ASSERT_EQUAL(4, count_consecutive_spaces("    cmd"));
 
-CTEST_SETUP(command_table_redirection_tests)
-{
-    (void)data;
-};
+}
 
-CTEST_TEARDOWN(command_table_redirection_tests)
+CTEST(replace_with_whitespace, basic_test)
 {
-    (void)data;
-};
+    char *str = ft_strdup("remove thisSuccess!");
+    char exp[] = "           Success!";
+    replace_redirection_w_space(&str, 11, 0);
+    ASSERT_STR(exp, str);
+    free(str);
+}
+
```

Comment:
why did you comment the tests?
If it is something that you have to implement yet or fix instead of CTEST use CTEST_SKIP